pkg/consensus/change_view.go

```diff
@@ -0,0 +1,18 @@
+package consensus
+
+import "github.com/CityOfZion/neo-go/pkg/io"
+
+type changeView struct {
```

Comment:
good to know a bit about it as well. At least to know why you choose such naming here.